test/update-coveralls

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Send coverage data to coveralls.io
+#
+# `go test -coverprofile=` only supports generating a coverage profile for one
+# package at a time. This script combines coverage data from multiple packages
+# and then sends the coverage data to coveralls.io
+#
+echo "mode: set" > acc.out
+FAIL=0
+
+packages="$(find . -type d -maxdepth 3 \
+	! -path './.git*' \
+	! -path '*/_*' \
+	! -path './Godeps*' \
+	! -path './test*' \
```

Comment:
As I read the PR it seems to me that the tests that are in the `./test` directory are not run anymore since this script replaced `go test ./...` from the `Makefile`. I think we still want to run the tests in the `./test` directory, but not collect the coverage report.